resolwe_bio/tools/build_species_mapper.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Map species to genome build."""
+import argparse
+
+parser = argparse.ArgumentParser(description="Map species to genome build")
+parser.add_argument('build', help="Genome build")
+args = parser.parse_args()
+
+mapper = {
+    'dm6': 'Drosophila melanogaster',
+}
+
+print(mapper[args.build])
```

Comment:
When this will get `build` value that is not present in the `mapper` it will fail with `KeyError`. It would be nice if you could raise a more descriptive error (preferably) or provide a default value (do we want that?).